### PR TITLE
Сreate urdf for stereo camera rotation

### PR DESCRIPTION
**Pull request by SystemDiagnosticss**
Сreate urdf for stereo camera rotation

### CODE REVIEW

**Comment by AndriyPt** [comment]
"Eyes" are floating in the air but should be on the "head" 
image

**Comment by AndriyPt** [comment]
@SystemDiagnosticss please also add `head_link` to Gazebo configuration. The same way as `base_link` was added here https://github.com/ros-ukraine/leobot/blob/kinetic-devel/leobot_description/robot/leobot.gazebo.xacro#L13-L19

**Comment by SystemDiagnosticss** [comment]
Done

**Comment by AndriyPt** [comment]
@SystemDiagnosticss from Gazebo I am getting this warning
```
[ WARN] [1518031419.419065884, 0.396000000]: Deprecated syntax, please prepend 'hardware_interface/' to 'EffortJointInterface' within the <hardwareInterface> tag in joint 'head_link_joint'.
``` 
Please fix it :)

File: leobot_description/robot/robot.urdf.xacro [hunk 1]
```diff
@@ -15,6 +15,35 @@
...
+    <xacro:cylinder_inertia mass="1" radius="0.15" height="0.1"/>
+  </link>
+
+  <joint name="head_link_joint" type="revolute">
+    <origin rpy="0 0 0" xyz="0 0 0.525"/>
+    <axis xyz="0 0 1" />
+    <parent link="base_link"/>
+    <child link="head_link"/>
+    <limit effort="10" velocity="10" lower="${-pi/2}" upper="${pi/2}" />
+    <dynamics damping="10" friction="1" />
```
Comment (AndriyPt) [inline leobot_description/robot/robot.urdf.xacro]: Please remove dynamics for now

**Review by AndriyPt — CHANGES_REQUESTED** [review]

**Review by AndriyPt — APPROVED** [review]

**Pull request merged**